homeassistant/components/zwave_me/config_flow.py

```diff
@@ -0,0 +1,88 @@
+"""Config flow to configure ZWaveMe integration."""
+
+from dataclasses import asdict
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import CONF_TOKEN, CONF_URL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ZWaveMeConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """ZWaveMe integration config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Initialize flow."""
+        self.url = vol.UNDEFINED
+        self.token = vol.UNDEFINED
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        errors = {}
+
+        if self._async_current_entries():
+            return self.async_abort(reason="already_configured")
+
+        if user_input is not None:
+            if "host" in user_input:
+                self.url = user_input["host"]
+
+            else:
+                if "url" in user_input:
+                    self.url = user_input["url"]
+                else:
+                    user_input["url"] = self.url
+                self.token = user_input["token"]
+
+                if not user_input["url"].startswith("ws://") and not user_input[
+                    "url"
+                ].startswith("wss://"):
```

Comment:
startswith takes a tuple
```suggestion
                if not user_input["url"].startswith(("ws://", "wss://")):
```